package-lock.json

```diff
@@ -2596,11 +2596,13 @@
         },
         "balanced-match": {
           "version": "1.0.0",
-          "bundled": true
+          "bundled": true,
+          "optional": true
```

Comment:
@ZachOrr These were automatically changed when I did a `npm install`.  I'm not sure if the change is necessary or not; I can take it out if you don't think we need it.

Reply:
Let's not checkin these changes

Reply:
Removed.